.gitignore

```diff
@@ -50,3 +50,6 @@ _sandbox
 env
 venv
 .python-version
+
+# IDE
+.idea/
```

Comment:
we need this folder in .gitignore file, because it's used in JetBrains IDEs, for example PyCharm

Reply:
This belongs in your [global gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).